public/wp-content/themes/newsup/readme.txt

```diff
@@ -159,4 +159,11 @@ License: CC0 Public Domain
 * Fixed sanitize issue.
 
 = Version 2.6
-* Added Edit link and fix archive page title issue.
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Merge errors to be removed.

This file isn't really relevant so it's not necessary, but still good for future note.

Reply:
oh